moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -68,13 +68,15 @@ class MoveitCpp
                planning_scene_monitor::PlanningSceneMonitor::MONITORED_PLANNING_SCENE_TOPIC);
       nh.param(ns + "publish_planning_scene_topic", publish_planning_scene_topic,
                planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_TOPIC);
+      nh.param<double>(ns + "wait_for_initial_state_timeout", wait_for_initial_state_timeout, 0.0);
```

Comment:
For future reference it would be great if you could create separate commits for unrelated contributions. For instance, adding this parameter, introducing MoveItErrorCode and changing the JointModelGroup member to raw pointer don't really belong to the unit tests.